test/unit/knex.js

```diff
@@ -223,6 +223,26 @@ describe('knex', () => {
     });
   });
 
+  it('supports direct retrieval of a transaction without a callback', (done) => {
+    const knex = Knex(sqliteConfig);
```

Comment:
Integration test that is ran for every dialect would be good to have to make sure that returned `trx` is immediately ready for making queries.

Reply:
Done